src/authors/scss/_body.scss

```diff
@@ -0,0 +1,68 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+body{
+ 
+    font-family: $family-font sans-serif;
+     
+     
+      background-color: rgb(51, 51, 88);
+      h1{
+        text-align: center;
+        margin: 30px;
+        font-size: 100px;
+
+      }
+      .wrap{
+        display: flex;
+        border-radius: 0 0 40px 40px;
+        margin: 30px;
+        padding:30px;
+        width: 100%;
+        align-items: center;
+        justify-content: center;
+
+        .box{
+            border: 2px solid black;
+                background-color: lightblue;
+                font-size: 1.5rem;
```

Comment:
Don't style in `rem` and `px`! Choose only one and in this project, we choose `px`.